src/components/ShowArtwork.astro

```diff
@@ -33,9 +33,20 @@ const { image } = Astro.props;
 </a>
 
 <script>
```

Comment:
```suggestion
<script type="module">
```
this essentially does the same thing, allows deferred non-blocking initialization

Reply:
Ooh, best not to do this! Astro will bundle this script and make it a module for you. But if you add `type="module"` yourself, Astro assumes you're in charge and doesn't process it at all, which would break the import.